histomicstk/utils/sample_pixels.py

```diff
@@ -86,14 +87,19 @@ def sample_pixels(slide_path, sample_fraction=None, magnification=None,
     iter_args = dict(scale=dict(magnification=magnification),
                      format=large_image.tilesource.TILE_FORMAT_NUMPY)
 
-    im_fgnd_mask_lres = dask.delayed(im_fgnd_mask_lres)
+    c = dask.distributed.get_client()
```

Comment:
@manthey This throws an exception if no client has been setup!

Should i guard it with a try catch block so it scatters only if client has been setup?

Or would we want it to throw an exception if no client has been setup